### PR TITLE
Update tasks datatable to use POST method

### PR DESCRIPTION
**Pull request by johnarnold**
This PR is to resolve https://github.com/mher/flower/issues/733

"flower can generate URLs which are too big for proxies "

### CODE REVIEW

File: flower/views/tasks.py [hunk 1]
```diff
@@ -83,6 +83,40 @@ def key(item):
...
+            task_dict = as_dict(self.format_task(task)[1])
+            if task_dict.get('worker'):
+                task_dict['worker'] = task_dict['worker'].hostname
+
+            filtered_tasks.append(task_dict)
+
+        self.write(dict(draw=draw, data=filtered_tasks,
+                        recordsTotal=len(sorted_tasks),
+                        recordsFiltered=len(sorted_tasks)))
+
```
Comment (mher) [inline flower/views/tasks.py]: Looks like `get` and `post` functions have identical body. It would be better to define `post` function with `post = get`.

Comment (johnarnold) [reply flower/views/tasks.py]: Will do. thanks.

Comment (johnarnold) [reply flower/views/tasks.py]: Done.

Comment (mher) [reply flower/views/tasks.py]: Thanks @johnarnold . I will merge this PR after fixing the tests.

**Pull request merged**